waspc/data/Generator/templates/react-app/src/queries/index.js

```diff
@@ -1,4 +1,5 @@
 import { useQuery as rqUseQuery } from 'react-query'
+export { setupQueryClient } from '../queryClient'
```

Comment:
@Martinsos What's the most appropriate place to expose this to our users? I'm not really happy with this one. Should I create something new like `@wasp/setup`?

Reply:
Ok, it can stay in `@wasp/queries` then. My main concern comes from the `QueryClient` constructor being used to set up something other than query behavior (e.g., mutations).

Reply:
Hm, that is a good point. 🤔 If used in actions, that may be a bit confusing. Yeah, perhaps in that case maybe pulling out of queries would make the most sense (or also re-exposing from actions, not sure if that is confusing then though).

Reply:
I don't think re-exposing solves our problem. After all, you can only import and configure it once, for both actions and queries at the same time. In other words, we would have two ways of importing/using it, none of which would be ideal.

Reply:
Yeah true, it could be more confusing. Pulling out into some new namespace makes sense to me to solve for this.

Reply:
Good point @sodic -> it is really the naming of react-query that is problem here :D. Or their naming of queryClient :D.
Let's scope it under `operations` then?

Reply:
We decided to scope it under `@wasp/queries` after all because:
- Operations are a private module
- We currently only use the QueryClient for queries (actions don't use _React Query_)